addons/binding/org.openhab.binding.yamahareceiver/src/main/java/org/openhab/binding/yamahareceiver/internal/state/AvailableInputState.java

```diff
@@ -0,0 +1,22 @@
+/**
+ * Copyright (c) 2010-2017 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.yamahareceiver.internal.state;
+
+import java.util.Map;
+import java.util.TreeMap;
+
+/**
+ * List of AVR input channel names with <Input ID, Input Name>
+ *
+ * @author David Graeff - Initial contribution
+ */
+public class AvailableInputState {
+    // List of inputs with <Input ID, Input Name>
+    public Map<String, String> availableInputs = new TreeMap<>();
+}
```

Comment:
@martinvw where are the new lines missing? The line ends have been added to the file end with my previous commit.